test/processors/concerns/http_files_test.rb

```diff
@@ -21,15 +21,25 @@ def logger
     WebMock.allow_net_connect!
   }
 
-  let(:uri) { URI.parse('http://test.prx.org/test/file.mp2') }
+  let(:http_uri) { URI.parse('http://test.prx.org/test/file.mp2') }
+  let(:https_uri) { URI.parse('https://test.prx.org/test/file.mp2') }
 
   it 'can download from http url' do
 
-    stub_request(:get, uri.to_s).
-      with(headers: {'Host'=>'test.prx.org:80'}).
+    stub_request(:get, http_uri.to_s).
+      with(headers: { 'Host' => 'test.prx.org:80' } ).
```

Comment:
Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.